core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -36,7 +36,7 @@ object UserService {
 
   case class OverwriteGroupMembers(groupRef: RawlsGroupRef, memberList: RawlsGroupMemberList)
 
-  def getGoogleProjectOwnerGroupEmail(samDAO: SamDAO, projectName: RawlsBillingProjectName)(implicit ec: ExecutionContext): Future[WorkbenchEmail] = {
+  def syncAndGetBillingProjectOwnerGroupEmail(samDAO: SamDAO, projectName: RawlsBillingProjectName)(implicit ec: ExecutionContext): Future[WorkbenchEmail] = {
```

Comment:
> gpolumbo-broad 2 hours ago • 
>  Member
> @andy7i - Oof, glad you caught this, this is why good method naming is so important. At the same time, I might suggest renaming this further. The primary thing this thing does is sync the policy with Google, and the name should reflect that.
> 
> The way it is named now (and before) it's like naming a side-effecting method for its return type. Imagine if you had a function that transferred money from your bank account to mine and returns the balance of your bank account, but you named the method getAccountBalance.
> 
> I would rather rename this to syncPolicyToGoogle, and it happens to return the email address of the synced policy instead of the "full" syncPolicy response object.


copied over from
https://github.com/broadinstitute/rawls/commit/54bfa41829090538d2b7d6d2b44473996ee8c075#r44994428

Reply:
we discussed a bit further and maybe what we want is `syncBillingProjectOwnerPolicyToGoogleAndGetEmail`? It's a bit of a mouthful so I'm open to other ideas here